feat(content): Petervdmeer Person Ship (#10374)
endless-sky/endless-sky: data/persons.txt

		word
			" "
		word
			"Arrakis"
			"Caladan"
			"the Eaxaw"
			"the Gyisache"
			"the Happy Birthday"
			"the Laenfa"
			"the Muursh"
			"the Ourbools"
			"Tycho Magnetic Anomaly 2"
		word
			"; "
		word
			"is this the right game?"
	phrase
		word
			"The Hauler VI is a nice ship, "
		word
			"as long as you don't activate the cargo refrigeration systems."
			"but it hasn't gotten beyond the prototype stage yet."
			"but it is not for sale on the regular market."
			"but using two Hauler IIIs is probably more economical."
	phrase
		word
			"Automate"
			"Extend"
			"Improve"
			"Streamline"
		word
			" "
		word
			"our"
			"the ES"
			"your"
		word
			" "
		word
			"builds"
			"builds and tests"
			"CI jobs"
			"coding style checks"
			"integration processes"
			"merge process"
			"software tests"
			"test and integration processes"
			"test processes"
			"tests"
		word
			"."
			"!"
		word
			" "
		word
			"Avoid crashes of the game!"
			"Improve the quality of our codebase!"
			"Get new features into the game faster!"
			"Have more time for good reviews!"
			"Have more time for playing the game!"
			"Prevent bugs!"

	ship "Modified Hauler VI" "PTR - HVI 42"

> 	ship "Hauler III" "PTR - Cliff"

Reviewer: Person esort-ships are typically restored after they get destroyed (if the flagship survives). What happens if one of those escorts gets captured, while the flagship isn't destroyed? Would the restore just move a captured ship back to the author government?

Reviewer: Just tested this; captured ships are indeed set back to the author government in this situation (giving the situation where your escorts suddenly become author-ships, and obviously attacking you, since the capture wasn't appreciated.)

Reviewer: Saving and reloading the game fixes this by creating a copy, but the issue still exists when you don't do a reload between capturing and encountering the author ship again.

Reviewer: I created an issue for this: https://github.com/endless-sky/endless-sky/issues/10379. We might want to make all the escort ships `"never captured"` for now though.

Reviewer: I applied the `"never disabled"` workaround for now on those ships. (Which means that they last a little bit longer in battle, but it shouldn't make much of a difference.)

	ship "Hauler III" "PTR - Sea"
	ship "Hauler II" "PTR - Rock"
	ship "Hauler II" "PTR - More"
	ship "Hauler" "PTR - Stone"
	ship "Hauler" "PTR - Lake"

ship "Modified Hauler VI"
	sprite "ship/hauler vi"
	thumbnail "thumbnail/hauler vi"

> 	"never disabled"

Reviewer: This `"never disabled"` keyword here doesn't seem to work; I just saw a disabled Hauler VI person ship. (While trying to capture its escort for the other remark I just made.)

Reviewer: I solved this by moving "never disabled" to the ships keywords directly under person. (Since it apparently is not carried over.)

			"interceptor"
			"fighter"
		word
			" "
		word
			"bays"
			"docking ports"
		word
			" "
		word
			"just for the fun of it."
			"for faster deployments."
			"on your new Marauder ship."
			"to improve atmospheric handling."
	phrase
		word
			"I have a delivery for"
			"I'm looking for"
		word
			" "
		word
			"Arrakis"
			"Caladan"
			"the Eaxaw"
			"the Gyisache"
			"the Happy Birthday"
			"the Laenfa"
			"the Muursh"
			"the Ourbools"
			"Tycho Magnetic Anomaly 2"
		word
			"; "
		word
			"is this the correct game?"
	phrase
		word
			"The Hauler VI is a nice ship, "
		word
			"as long as you don't activate the cargo refrigeration systems."
			"but it didn't get beyond the prototype stage yet."
			"but it is not for sale on the regular market."
			"but using two Hauler IIIs is probably more economical."
	phrase
		word
			"Automate"
			"Extend"
			"Improve"
			"Streamline"
		word
			" "
		word
			"our"
			"the ES"
			"your"
		word
			" "
		word
			"builds"
			"builds and tests"
			"ci jobs"
			"coding style checks"
			"integration processes"
			"merge process"
			"software tests"
			"test and integration processes"
			"test processes"
			"tests"
		word
			"."
			"!"

> 		word

Reviewer: ```suggestion
		word
			" "
		word
```

Reviewer: (Found during testing of #10377)

	thumbnail "outfit/unknown"
	description `The physical embodiment of everything that makes the Reviewers who they are today.`
	description `	Unfortunately, it's not for sale.`

person "Danau"
	government "Author"
	frequency 100
	personality
		heroic unconstrained opportunistic
	phrase
		word

> 			"When you want to stop your ship, then press shift and back."

Reviewer 2: This sentence confuses me. I get what it's trying to say, but I'm not sure it's working.

`then` would work better with `if` than with `when`. You could probably remove it entirely.

Back isn't a key, but it makes sense in case as a verb if you've rebound your keys. Still, I'd probably go with "reverse" or similar, since that use of back is very niche.

```suggestion
			"When you want to stop your ship, press shift and reverse."
```

			"Your hyperdrive doesn't work? Have you tried turning it off and on again?"
	phrase
		word
			"Change the orientation of your"
		word
			" "
		word
			"drone"
			"interceptor"
			"fighter"
		word
			" "
		word
			"bays"
			"docking ports"
		word
			" "
		word
			"just for the fun of it."
			"for faster deployments."
			"on your new Marauder ship."
			"to improve atmospheric handling."
	phrase
		word
			"I have a delivery for"
			"I'm looking for"
		word
			" "
		word
			"Arrakis"
			"Caladan"
			"the Eaxaw"
			"the Gyisache"
			"the Happy Birthday"
			"the Laenfa"
			"the Muursh"
			"the Ourbools"
			"Tycho Magnetic Anomaly 2"
		word
			"; "
		word

> 			"is this the correct game?"

Reviewer 2: This definitely works, but I don't usually hear people say, "is this the correct ..."—usually it's "is this the right..."

You can keep it as is, or:

```suggestion
			"is this the right game?"
```

or both:

```suggestion
			"is this the correct game?"
			"is this the right game?"
```

	phrase
		word
			"The Hauler VI is a nice ship, "
		word
			"as long as you don't activate the cargo refrigeration systems."

> 			"but it didn't get beyond the prototype stage yet."

Reviewer 2: ```suggestion
			"but it hasn't gotten beyond the prototype stage yet."
```

			"but it is not for sale on the regular market."
			"but using two Hauler IIIs is probably more economical."
	phrase
		word
			"Automate"
			"Extend"
			"Improve"
			"Streamline"
		word
			" "
		word
			"our"
			"the ES"
			"your"
		word
			" "
		word
			"builds"
			"builds and tests"

> 			"ci jobs"

Reviewer 2: The only other place in the game that CI is mentioned, it's capitalized.
```suggestion
			"CI jobs"
```

			"coding style checks"
			"integration processes"
			"merge process"
			"software tests"
			"test and integration processes"
			"test processes"
			"tests"
		word
			"."
			"!"
		word
			" "

> 		word
> 			"So that we"
> 			"So that you"
> 			"To"
> 		word
> 			" avoid crashes of the game."
> 			" improve the quality of our codebase."
> 			" get new features into the game faster."
> 			" have more time for good reviews."
> 			" have more time for playing the game."
> 			" prevent bugs."

Reviewer: ```suggestion
		word
			"Avoid crashes of the game!"
			"Improve the quality of our codebase!"
			"Get new features into the game faster!"
			"Have more time for good reviews!"
			"Have more time for playing the game!"
			"Prevent bugs!"
```

> 			" prevent bugs."

Reviewer 2: This isn't a valid sentence. It's like saying "Do this. So that we avoid crashes of the game." It _could_ be used anyways, but it's passive-aggressive and bad grammar.

I'd probably remove the punctuation at the end of the last sentence and the capitalization at the beginning of this one and merge the two into one. Alternatively, you could Just cut out the first block here and make phrases like "Automate our builds. Get new features into the game faster!"

Reviewer: Thanks! Good feedback. I chose the second option.

	thumbnail "outfit/unknown"
	description `The physical embodiment of everything that makes the Reviewers who they are today.`
	description `	Unfortunately, it's not for sale.`

person "Danau"
	government "Author"
	frequency 100
	personality

> 		heroic unconstrained opportunistic

Author: This is how all other persons do it.

	phrase
		word
			"When you want to stop your ship, press shift and reverse."
			"Your hyperdrive doesn't work? Have you tried turning it off and on again?"
	phrase
		word
			"Change the orientation of your"
		word
			" "
		word
			"drone"
			"interceptor"
			"fighter"
		word
			" "
		word
			"bays"
			"docking ports"
		word
			" "
		word
			"just for the fun of it."
			"for faster deployments."

> 			"on your new Marauder ship."

Author: I don't see why we have to limit what persons talk about what. Yes, they could have specific dialog on related things, but no reason to restrict what others say.

		word
			"I have a delivery for"
			"I'm looking for"
		word
			" "
		word
			"Arrakis"
			"Caladan"
			"the Eaxaw"
			"the Gyisache"
			"the Happy Birthday"
			"the Laenfa"
			"the Muursh"
			"the Ourbools"
			"Tycho Magnetic Anomaly 2"
		word
			"; "
		word
			"is this the right game?"
	phrase
		word
			"The Hauler VI is a nice ship, "
		word
			"as long as you don't activate the cargo refrigeration systems."
			"but it hasn't gotten beyond the prototype stage yet."
			"but it is not for sale on the regular market."
			"but using two Hauler IIIs is probably more economical."
	phrase
		word
			"Automate"
			"Extend"
			"Improve"
			"Streamline"
		word
			" "
		word
			"our"
			"the ES"
			"your"
		word
			" "
		word
			"builds"
			"builds and tests"
			"CI jobs"
			"coding style checks"
			"integration processes"
			"merge process"
			"software tests"
			"test and integration processes"
			"test processes"
			"tests"
		word
			"."
			"!"
		word
			" "
		word
			"Avoid crashes of the game!"
			"Improve the quality of our codebase!"
			"Get new features into the game faster!"
			"Have more time for good reviews!"
			"Have more time for playing the game!"
			"Prevent bugs!"

> 	ship "Modified Hauler VI" "PTR - HVI 42"

Author: My guess is that's the way Peter wants to have the ships named; PTR representing his name. More his design choice than anything.

> 	ship "Modified Hauler VI" "PTR - HVI 42"

Reviewer: I removed the prefixes, they indeed looked a bit out of place.

		"never disabled"
	ship "Hauler III" "PTR - Cliff"
		"never disabled"
	ship "Hauler III" "PTR - Sea"
		"never disabled"
	ship "Hauler II" "PTR - Rock"
		"never disabled"
	ship "Hauler II" "PTR - More"
		"never disabled"
	ship "Hauler" "PTR - Stone"
		"never disabled"
	ship "Hauler" "PTR - Lake"
		"never disabled"

ship "Modified Hauler VI"
	sprite "ship/hauler vi"
	thumbnail "thumbnail/hauler vi"
	attributes
		category "Heavy Freighter"
		"shields" 13200
		"hull" 26800

> 		"required crew" 3

Author: (These aren't capturable anyways, but yeah, same crew amounts across the Haulers I-III.)

> 		"required crew" 3

Reviewer: Haulers I, II and III all have "required crew" 3 and "bunks" 12, so I believe this is adressed.

		"bunks" 12
		"mass" 1240
		"drag" 21.7
		"heat dissipation" .43
		"fuel capacity" 800
		"cargo space" 780
		"outfit space" 900
		"weapon capacity" 430
		"engine capacity" 160
		weapon
			"blast radius" 100
			"shield damage" 1000
			"hull damage" 500
			"hit force" 1500
	outfits
		"Sidewinder Missile Launcher" 4
		"Sidewinder Missile" 180
		"Point Defense Turret" 2
		"Bombardment Turret" 6

		"Fusion Reactor"
		"S3 Thermionic"
		"LP288a Battery Pack"
		"D94-YV Shield Generator"
		"Large Radar Jammer"
		"Laser Rifle" 3
		
		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Capybara Reverse Thruster"
		"Hyperdrive"

	engine -83.5 196.5
	engine 83.5 196.5
	engine 0 196.5
	gun -14.5 -168
	gun 14.5 -168
	gun -35 -128
	gun 35 -128
	turret -21 -124.5
	turret 21 -124.5 "Point Defense Turret"
	turret -32.5 -102.5
	turret 32.5 -102.5
	turret -22 154 "Point Defense Turret"
	turret 22 154
	turret -60 154
	turret 60 154
	leak "leak" 70 50
	leak "flame" 40 80
	leak "big leak" 90 40
	explode "small explosion" 30
	explode "medium explosion" 60
	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "The Hauler cargo ships were considered by many to be outdated, and it is unclear if upgrading the cargo-space to six containers is sufficient to make this Hauler ship popular. It certainly doesn't help that the Hauler VI is more expensive than four Hauler IIIs."

Reviewer 3: ```suggestion
	description "The Hauler cargo ships were considered by many to be outdated, and it is unclear if upgrading the cargo space to six containers is sufficient to make this Hauler ship popular. It certainly doesn't help that the Hauler VI is more expensive than four Hauler IIIs."
```

			"fighter"
		word
			" "
		word
			"bays"
			"docking ports"
		word
			" "
		word
			"just for the fun of it."
			"for faster deployments."
			"on your new Marauder ship."
			"to improve atmospheric handling."
	phrase
		word
			"I have a delivery for"
			"I'm looking for"
		word
			" "
		word
			"Arrakis"
			"Caladan"
			"the Eaxaw"
			"the Gyisache"
			"the Happy Birthday"
			"the Laenfa"
			"the Muursh"
			"the Ourbools"
			"Tycho Magnetic Anomaly 2"
		word
			"; "
		word
			"is this the right game?"
	phrase
		word
			"The Hauler VI is a nice ship, "
		word
			"as long as you don't activate the cargo refrigeration systems."
			"but it hasn't gotten beyond the prototype stage yet."
			"but it is not for sale on the regular market."
			"but using two Hauler IIIs is probably more economical."
	phrase
		word
			"Automate"
			"Extend"
			"Improve"
			"Streamline"
		word
			" "
		word
			"our"
			"the ES"
			"your"
		word
			" "
		word
			"builds"
			"builds and tests"
			"CI jobs"
			"coding style checks"
			"integration processes"
			"merge process"
			"software tests"
			"test and integration processes"
			"test processes"
			"tests"
		word
			"."
			"!"
		word
			" "
		word

> 			"Avoid crashes of the game!"

Reviewer 4: ```suggestion
			"Avoid game crashes!"
```

		"hull" 26800
		"required crew" 3
		"bunks" 12
		"mass" 1240
		"drag" 21.7
		"heat dissipation" .43
		"fuel capacity" 800
		"cargo space" 780
		"outfit space" 900
		"weapon capacity" 430
		"engine capacity" 160
		weapon
			"blast radius" 100
			"shield damage" 1000
			"hull damage" 500
			"hit force" 1500
	outfits
		"Sidewinder Missile Launcher" 4
		"Sidewinder Missile" 180
		"Point Defense Turret" 2
		"Bombardment Turret" 6

		"Fusion Reactor"
		"S3 Thermionic"
		"LP288a Battery Pack"
		"D94-YV Shield Generator"
		"Large Radar Jammer"
		"Laser Rifle" 3
		
		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Capybara Reverse Thruster"
		"Hyperdrive"

	engine -83.5 196.5
	engine 83.5 196.5
	engine 0 196.5
	gun -14.5 -168
	gun 14.5 -168
	gun -35 -128
	gun 35 -128
	turret -21 -124.5
	turret 21 -124.5 "Point Defense Turret"
	turret -32.5 -102.5
	turret 32.5 -102.5
	turret -22 154 "Point Defense Turret"
	turret 22 154
	turret -60 154
	turret 60 154
	leak "leak" 70 50
	leak "flame" 40 80
	leak "big leak" 90 40
	explode "small explosion" 30
	explode "medium explosion" 60
	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "The Hauler cargo ships were considered by many to be outdated, and it is unclear if upgrading the cargo space to six containers is sufficient to make this Hauler ship popular. It certainly doesn't help that the Hauler VI is more expensive than four Hauler IIIs."

Reviewer 4: ```suggestion
	description "The Hauler cargo ships are considered by many to be outdated, and it is unclear if upgrading the cargo space to six containers is sufficient to make this Hauler ship popular. It certainly doesn't help that the Hauler VI is more expensive than four Hauler IIIs."
```
